student-projects/ravivos1/class14/calculator/cal.ts

```diff
@@ -0,0 +1,18 @@
+const students = Number (prompt("How many grades you want to enter?"))
+if (students <18 || isNaN (students) || students===null || students <= 0) {
+    alert("You didn't entered a valid number, please refresh the page and enter the number of grades you want to enter");
+}
+else {
+    for (let i = 1; i <= students; i++) {
+        const studentsleft = students - i;
+        const grade = Number(prompt("student number " + i + "\nStudents left " + studentsleft + "\nPlease enter the next grade"));
+        if (i===1) {
```

Comment:
`sum` should be defined before the loop.
It should be a `let` variable.
You can initialize it with 0 and then you don't need the "split" logic. Just always add the current number.